config/routes.rb

```diff
@@ -6,6 +6,7 @@
   end
   namespace :api do
     namespace :v1 do
+      resources :evaluations, only: [:index]
```

Comment:
please, move this in scope of profile. Evaluations are not something general. It is related to users profile